utils/geoareas_exif_integration.py

```
@@ -0,0 +1,435 @@
+        
+        # Check geo-areas fields (only those used for ExifTool integration)
+        geoareas_fields = [
+            "geo_place", "geo_county", "geo_state",  # Core EXIF tags
+            "geo_prev_place", "geo_prev_miles", "geo_next_place", "geo_next_miles"  # Context calculation
+        ]
+        
+        availability = {}
+        for field in geoareas_fields:
+            availability[field] = field.lower() in field_names
+        
+        return availability
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)